rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/security/Sw360SecurityEncryptor.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Siemens AG, 2018. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.sw360.rest.authserver.security;
+
+import org.apache.log4j.Logger;
+
+import javax.crypto.*;
+import javax.crypto.spec.SecretKeySpec;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+public class Sw360SecurityEncryptor {
```

Comment:
Encrypting with a hard coded password is only obfuscation? Maybe the class could better be called `Sw360SecurityObfuscator`? :wink:

Reply:
hmm.. I am absolutely not familiar with `Sw360SecurityObfuscator` :hear_no_evil: